src/herder/TransactionQueue.cpp

```diff
@@ -302,7 +300,7 @@ TransactionQueue::tryAdd(TransactionFrameBasePtr tx)
     auto ops = tx->getNumOperations();
     stateIter->second.mQueueSizeOps += ops;
     stateIter->second.mBroadcastQueueOps += ops;
-    mQueueSizeOps += ops;
+    mLimiter.removedTransaction(tx);
```

Comment:
Sorry, I did the same thing to Graydon, this is fixed in the next commit.

That first commit was true garbage, probably created it way too early: I had all possible typos and bugs possible.
I'll fixup this commit to avoid further confusion for a future observer.